src/utils/indexFile.ts

```diff
@@ -0,0 +1,22 @@
+import path from 'path';
+import normalizePath from 'normalize-path';

```

Comment:
Do we really need to add a new dependency? Can we do without it using native Nodejs?